.github/workflows/build_addon.yml

```diff
@@ -0,0 +1,271 @@
+name: Build Addon
+
+on: [push, pull_request]
+
+env:
+  godot-version: 4.2.1-stable
+
+jobs:
+  # Build the Windows Artifacts
+  build-windows:
+    name: 🏁 Build Windows
+    runs-on: windows-2019
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.9"
+
+      - name: Setup Bazelisk
+        uses: bazelbuild/setup-bazelisk@v3
+
+      - name: Prerequisites
+        shell: bash
+        run: |
+          choco install visualstudio2019buildtools visualstudio2019-workload-vctools
+          choco install opencv --version 3.4.10
+
+      - name: Download Godot
+        run: |
+          curl -L -O https://github.com/godotengine/godot/releases/download/${{ env.godot-version }}/Godot_v${{ env.godot-version }}_win64.exe.zip
+          unzip -o Godot_v${{ env.godot-version }}_win64.exe.zip -d godot
+
+      - name: Setup
+        shell: bash
+        run: |
+          python ./setup.py --godot-binary ./godot/Godot_v${{ env.godot-version }}_win64.exe --custom-opencv-dir C:/tools/opencv/build
+
+      - name: Build
+        env:
+          BAZEL_VC: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\BuildTools\\VC"
+        shell: bash
+        run: |
+          source ./venv/Scripts/activate
+          python ./build.py desktop --type release
+
+      - name: Upload build files (artifacts)
+        uses: actions/upload-artifact@v4
+        with:
+          name: build-artifact-windows
+          path: |
+            mediapipe/bazel-bin/GDMP/desktop/*.dll
+
+
+  # Build the Linux Artifacts
+  build-linux:
+    name: 🐧 Build Linux
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.9"
+
+      - name: Setup Bazelisk
+        uses: bazelbuild/setup-bazelisk@v3
+
+      - name: Prerequisites
+        run: |
+          sudo apt-get install -y \
+            ffmpeg \
+            libegl1-mesa-dev \
+            libopencv-core-dev \
+            libopencv-highgui-dev \
+            libopencv-calib3d-dev \
+            libopencv-features2d-dev \
+            libopencv-imgproc-dev \
+            libopencv-video-dev \
+            libopencv-contrib-dev
```

Comment:
Not a problem but concern: by linking libraries provided by specific distribution like ubuntu 22.04, released GDMP may not work if the target use different environments.
Ultimately my goal is to build GDMP on distro-independent runtime (e.g. Freedesktop platform in Flatpak) for linux library. But that involves making some changes to mediapipe workspace, so let's build linux library this way for now.

Reply:
Sounds good - I'm sure the CI actions will go through much improvement.